Documentation/tutorials/EdgeBridgeTutorialAppStart/EdgeBridgeTutorialApp.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -0,0 +1,68 @@
+{
+  "pins" : [
+    {
+      "identity" : "aepsdk-analytics-ios",
+      "kind" : "remoteSourceControl",
+      "location" : "https://github.com/adobe/aepsdk-analytics-ios.git",
+      "state" : {
+        "branch" : "main",
+        "revision" : "c6866d8989686e532a528c508bf74eca8b5efe57"
+      }
+    },
+    {
+      "identity" : "aepsdk-assurance-ios",
+      "kind" : "remoteSourceControl",
+      "location" : "https://github.com/adobe/aepsdk-assurance-ios.git",
+      "state" : {
+        "branch" : "main",
+        "revision" : "55c6419494d9cb88774a6a382d4b72756af95900"
+      }
+    },
+    {
+      "identity" : "aepsdk-core-ios",
+      "kind" : "remoteSourceControl",
+      "location" : "https://github.com/adobe/aepsdk-core-ios.git",
+      "state" : {
+        "branch" : "main",
+        "revision" : "255e3fde7c478a48c57e3f6ee9df1700aae22278"
+      }
+    },
+    {
+      "identity" : "aepsdk-edge-ios",
```

Comment:
I think the start app should not have these dependencies yet, right? only after the tutorial steps you will add them

Reply:
nvm, after reading more through I think this is correct for the start app, you just add edgrebridge later

Reply:
Should I keep the resolved file there?